pxr/usd/usdSemantics/schema.usda

```diff
@@ -0,0 +1,35 @@
+#usda 1.0
+(
+    "This describes the schemas for semantic labels."
+    subLayers = [
+        @usd/schema.usda@
+    ]
+)
+
+over "GLOBAL" (
+    customData = {
+        string libraryName = "usdSemantics"
+        string libraryPath = "pxr/usd/usdSemantics"
+    }
+) {
+}
+
+class "SemanticsLabelsAPI" (
+    inherits = </APISchemaBase>
+    doc = """Applied set of labels for a prim for a taxonomy specified by the
```

Comment:
Given implied meanings of "set", can we be specific on what we mean, here?  Is it an ordered list of labels, or is authored order insignificant?